Floofbot/Migrations/20200501123654_AddNicknameAlertAndRenameCols.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Floofbot.Migrations
+{
+    public partial class AddNicknameAlertAndRenameCols : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.RenameColumn(
+                name: "UserID",
+                table: "Tags",
+                newName: "UserId");
+
```

Comment:
Does the migration really need to modify the tag table?

Reply:
The migration generates updates based on the names of the fields, so if the name of the field was UserID, but is now UserId, then it'll rename the Database.

Reply:
have you experienced any issues with the other modules as a result of the changes?